README.md

```diff
@@ -160,6 +160,7 @@ If you're using the default tenant, the URL is `ws://realtime-dev.localhost:4000
 | JANITOR_CLEANUP_CHILDREN_TIMEOUT     | number  | Timeout for each async task for janitor cleanup                                                                                                                                                                                                                                                                                 |
 | JANITOR_CHUNK_SIZE                   | number  | Number of tenants to process per chunk. Each chunk will be processed by a Task                                                                                                                                                                                                                                                  |
 | MIGRATION_PARTITION_SLOTS            | number  | Number of dynamic supervisor partitions used by the migrations process                                                                                                                                                                                                                                                          |
+| CONNECT_PARTITION_SLOTS              | number  | Number of dynamic supervisor partitions used by the Connect, ReplicationConnect and Listen processes                                                                                                                                                                                                                            |
```

Comment:
@filipecabaco we were both right! The env already exists but it was not documented:

https://github.com/supabase/realtime/blob/f2bef3168eebea212273a1745dfa42736f78b6c5/config/runtime.exs#L21-L22